spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/ComposedTaskRunnerConfigurationPropertiesTests.java

```diff
@@ -43,7 +44,16 @@ void useUserAccessTokenFromCTRPropFalse() {
 	void useUserAccessTokenFromCTRPropNotSet() {
 		ComposedTaskRunnerConfigurationProperties composedTaskRunnerConfigurationProperties =
 			new ComposedTaskRunnerConfigurationProperties();
-		assertThat(composedTaskRunnerConfigurationProperties.isUseUserAccessToken()).as("Use user access token should be false").isNull();
+		assertThat(composedTaskRunnerConfigurationProperties.isUseUserAccessToken()).as("Use user access token should be false").isFalse();
```

Comment:
I have found the `as` and also Junit `describe` end up being another maintenance point that comes out of date very quickly (i.e. they are more hassle than they are worth IMO). It was here already, just saying this out loud though.